UserDetailsClient/UserDetailsClient.Core/App.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Microsoft.Identity.Client;
+using UserDetailsClient.Core.Features.LogOn;
+
+[assembly: XamlCompilation(XamlCompilationOptions.Compile)]
+namespace UserDetailsClient.Core
+{
+	public partial class App : Application
+    {
+
+        public static string ApiEndpoint = "https://fabrikamb2chello.azurewebsites.net/hello";
+
+        public App ()
+		{
```

Comment:
Please format all code files

Reply:
code looks properly tabbed to me tbh

![image](https://user-images.githubusercontent.com/3155241/57165502-47acc280-6dc5-11e9-9af5-f4bbfac310c3.png)